solr/core/src/java/org/apache/solr/update/UpdateLog.java

```diff
@@ -268,40 +268,13 @@ public String toString() {
   protected Meter copyOverOldUpdatesMeter;
   protected SolrMetricsContext solrMetricsContext;
 
-  public static class LogPtr {
-    final long pointer;
-    final long version;
-    // used for entries that are in-place updates and need a pointer to a previous update command
-    final long previousPointer;
```

Comment:
This was in here since when it first showed up like 10 years ago, and even then wasn't actually *used*.  Populated but not referenced.  CC @chatman